certbot-nginx/certbot_nginx/constants.py

```diff
@@ -17,6 +17,9 @@
 """Path to the nginx mod_ssl config file found in the Certbot
 distribution."""
 
+UPDATED_MOD_SSL_CONF_DIGEST = ".updated-mod-ssl-conf-digest.txt"
```

Comment:
We may want to change the name of the file here as I can see it conflicting with what we do for Apache.